test/account/paymaster/Paymaster.behavior.js

```diff
@@ -92,11 +92,17 @@ function shouldBehaveLikePaymaster({ postOp, timeRange }) {
                 }),
               ]),
             })
-            .then(op => this.paymasterSignUserOp(op, { validAfter: MAX_UINT48 })) // validAfter MAX_UINT48 is in the future
+            .then(op =>
+              this.paymasterSignUserOp(op, {
+                // validAfter MAX_UINT48 is in the future
+                // shr by 1 to remove the most significant bit, indicating block ranges
+                validAfter: MAX_UINT48 >> 1n,
+              }),
```

Comment:
```suggestion
              this.paymasterSignUserOp(op, {
                // validAfter MAX_UINT48 is in the future
                // shr by 1 to remove the most significant bit, indicating timestamp ranges
                validAfter: MAX_UINT48 >> 1n,
              }),
```

Setting the highest bit indicates block ranges, removing it indicates timestamp ranges. [source](https://github.com/eth-infinitism/account-abstraction/releases/tag/v0.9.0#:~:text=Now%2C%20setting%20the%20highest%20bit%20of%20both%20validAfter%20and%20validUntil%20parameters%20indicates%20to%20bundlers%20and%20the%20EntryPoint%20that%20these%20values%20specify%20the%20UserOperation%E2%80%99s%20validity%20range%20using%20block%20numbers.)